tox.ini

```diff
@@ -154,6 +154,7 @@ commands = python scripts/publish-gh-release-notes.py {posargs}
 
 [flake8]
 max-line-length = 120
+select = E,F,W,C,TYP,D200,D201,D206,D207,D208,D210,D211,D212,D214,D215,D300,D301,D403,D404
```

Comment:
pydocstyle has an [extensive list](http://www.pydocstyle.org/en/stable/error_codes.html) of error codes. Most of them we don't want, and some have too many false-positives. So I opted to only select the error codes we want instead of excluding those we don't want.

I'm not sure if this is the best way to achieve this -- suggestions welcome.

Reply:
in general, extend-ignore is preferred over `select`

Reply:
OK, updated to use extend-ignore.